### PR TITLE
Updating updated at

### PR DESCRIPTION
**Pull request by andrewconner**
@stkem 

This problem may exist elsewhere, but it's a start

### CODE REVIEW

File: kifi-backend/modules/eliza/app/com/keepit/eliza/UserThreadRepo.scala [hunk 1]
```diff
@@ -179,28 +179,28 @@ class UserThreadRepoImpl @Inject() (
...
     Query(table).filter(row => (row.user===userId && row.thread===threadId) && row.lastMsgFromOther === message.id.get)
-      .map(row => row.lastNotification ~ row.notificationEmailed)
-      .update((notifJson, false))
+      .map(row => row.lastNotification ~ row.notificationEmailed ~ row.updatedAt)
+      .update((notifJson, false, clock.now()))
   }
 
   def setLastSeen(userId: Id[User], threadId: Id[MessageThread], timestamp: DateTime)(implicit session: RWSession) : Unit = {  //Note: minor race condition
-    (for (row <- table if row.user===userId && row.thread===threadId && (row.lastSeen < timestamp || row.lastSeen.isNull)) yield row.lastSeen).update(timestamp)
+    (for (row <- table if row.user===userId && row.thread===threadId && (row.lastSeen < timestamp || row.lastSeen.isNull)) yield row.lastSeen ~ row.updatedAt).update((timestamp, clock.now()))
```
Comment (2is10) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/UserThreadRepo.scala:203]: I’m not sure we should be updating `updatedAt` in this case. This `lastSeen` time wasn’t originally stored on a per-thread basis (in shoebox).

Comment (2is10) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/UserThreadRepo.scala:203]: Nevermind. Glad we aren’t using `updatedAt` for any logic.

**Comment by stkem** [comment]
lgtm

**Comment by andrewconner** [comment]
@ymatsuda

**Pull request merged**